index.html

```diff
@@ -248,8 +239,19 @@ <h1>About <br />Myself</h1>
         </ul>
         <div class="dropdown"><li class="section_header">Frameworks</li><span>^</span></div>
         <div class="dropdown"> <li class="section_header">Skills</li><span>^</span>
-       
       </ul>
     </section>
+    <footer class="footer">
+        <form action="https://formspree.io/f/mjvzpbgj" method="post">
+            <h1 class="form_header">Contact me</h1>
+            <p class="contact_paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
```

Comment:
- [OPTIONAL]  I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. 
for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/). 💡